packer/apt-upgrade.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+while sudo fuser -s /var/lib/dpkg/lock; do
```

Comment:
Can you please point me to the documentation of the exit status of this command? Cases: file not present, file present with no process having it open, file present with at lease a process having it open.

Reply:
I'm afraid it's not documented. I tested it myself. However I also found this for you: https://gitlab.com/psmisc/psmisc/blob/master/src/fuser.c#L1355

I also assume that the command will return non 0 exit code if the file does not exists, however we can assume it always exists because of base image usage (where it's always present).